project.php

```diff
@@ -2028,22 +2031,15 @@ function do_change_state(): void
             $text_next_to_btn = "<span class='transition-disabled'>$text_next_to_btn</span> [$reason]";
         }
 
-        echo "<form method='POST' action='$code_url/tools/changestate.php'>";
+        $question = attr_safe($transition->confirmation_question);
+        echo "<form method='POST' class='frm_chg_stt' data-confirmation-question='$question' action='$code_url/tools/changestate.php'>";
```

Comment:
I'd prefer the more verbose wording of the class -- a few extra characters to be clearer won't hurt.
```suggestion
        echo "<form method='POST' class='form_change_state' data-confirmation-question='$question' action='$code_url/tools/changestate.php'>";
```

Reply:
Done this.